tests/x11/firefox_audio.pm

```diff
@@ -22,7 +22,15 @@ use testapi;
 sub run {
     my ($self) = @_;
     start_audiocapture;
-    x11_start_program('firefox ' . data_url('1d5d9dD.oga'), target_match => 'test-firefox_audio-1', match_timeout => 90);
+    x11_start_program('firefox ' . data_url('1d5d9dD.oga'), target_match => [qw(command-not-found test-firefox_audio-1)], match_timeout => 90);
+    #  re-try for typing issue, see https://progress.opensuse.org/issues/54401
+    if (match_has_tag 'command-not-found') {
```

Comment:
You need to place this, inside a loop, and as we mentioned, retry up to 3 times, if `match_has_tag 'command-not-found'` return true repeat the operation, so you will need to call again x11_start_program, since `send_key 'esc'` will close the command window again, you will be typing nowhere.

Reply:
yes, thanks.

Reply:
@Zaoliang please trigger a verification run, scheduling the firefox module multiple times. 

I guess using the `SCHEDULE=$TEST_MODULES` parameter should suffice

Reply:
yes, over 30 times got triggered on O3:
https://openqa.opensuse.org/tests/1006387#next_previous

Reply:
@Zaoliang no, schedule within the same job, the firefox module 10-20 times :). So we see within the same job, firefox 30 times. That's what I meant with the `SCHEDULE=$TEST_MODULES` setting.

Reply:
https://openqa.opensuse.org/tests/1007421#next_previous
failed tests(incomplete) are related to https://progress.opensuse.org/issues/55505
some failed tests are related to typing issue which happened at firefox, see:
https://progress.opensuse.org/issues/55505